Shared/Extensions/URLExtensions.swift

```diff
@@ -192,6 +192,22 @@ extension URL {
             return urlString
         }
     }
+    
+    public var typedDisplayString: String {
```

Comment:
i assume the URL is already normalized to lowercase? might be good to have tests for this function

Reply:
Yes this function already start the processing with absoluteString for the URL. Tests are added at https://github.com/brave/brave-ios/pull/3703/commits/e5d686c76f61778b93219063462a2703f7ab2bf4

Please check https://github.com/brave/brave-ios/pull/3703#discussion_r668010909 for additional details